### PR TITLE
Profile Detail Page Without Integration #20

### PR DESCRIPTION
**Pull request by seyeonigbinde**
### What this PR does (required):
- Designed the UI of the Profile Detail Page #20 
- Setup DatePicker for the Drop In and Drop Off in Profile Detail Page #20 
- Created demo initialValues for all the db data #20 

### Screenshots / Videos (front-end only):

Screenshot 2021-10-29 at 01 59 04

### CODE REVIEW

File: client/src/pages/Profile/ProfileDetailPage/ProfileDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import DateFnsUtils from '@date-io/date-fns';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import Rating from '@material-ui/lab/Rating';
+import avatar from '../../../Images/68f55f7799df6c8078a874cfe0a61a5e6e9e1687.png';
+
+import useStyles from './useStyles';
+
+const theme = createMuiTheme();
+
+const initialValues = {
```
Comment (rajivtitus) [inline client/src/pages/Profile/ProfileDetailPage/ProfileDetail.tsx:16]: 👍🏽

File: client/src/pages/Profile/ProfileDetailPage/ProfileDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import { DateTimePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import Rating from '@material-ui/lab/Rating';
+import avatar from '../../../Images/68f55f7799df6c8078a874cfe0a61a5e6e9e1687.png';
+
+import useStyles from './useStyles';
+
+const theme = createMuiTheme();
```
Comment (rajivtitus) [inline client/src/pages/Profile/ProfileDetailPage/ProfileDetail.tsx:14]: Question: Why are we creating a new theme? The app already has a global theme applied

File: client/src/pages/Profile/ProfileDetailPage/ProfileDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+              <LocationOnIcon className={classes.locationLogo} />
+              {initialValues.location}
+            </Typography>
+          </Box>
+          <Box className={classes.description}>
+            <Typography className={classes.aboutTitle}>About Me</Typography>
+            <Typography className={classes.about}>{initialValues.description}</Typography>
+          </Box>
+          <Box className={classes.petName}>
+            {initialValues.dogs.map((item) => (
```
Comment (rajivtitus) [inline client/src/pages/Profile/ProfileDetailPage/ProfileDetail.tsx:68]: 👍🏽

**Review by rajivtitus — APPROVED** [review]
Great work with this! Just a question for you :)

**Comment by seyeonigbinde** [comment]
@rajivtitus , createMuiTheme and ThemeProvider have been deleted

**Comment by rajivtitus** [comment]
@seyeonigbinde The PR has been approved, please go ahead and merge it :)

**Pull request merged**